diff --git a/Contrastive_Learning/simclr1/__pycache__/model.cpython-39.pyc b/Contrastive_Learning/simclr1/__pycache__/model.cpython-39.pyc
index 893319b..0af4f06 100644
Binary files a/Contrastive_Learning/simclr1/__pycache__/model.cpython-39.pyc and b/Contrastive_Learning/simclr1/__pycache__/model.cpython-39.pyc differ
diff --git a/Contrastive_Learning/simclr1/__pycache__/utils.cpython-39.pyc b/Contrastive_Learning/simclr1/__pycache__/utils.cpython-39.pyc
index aae26fc..e6f14b2 100644
Binary files a/Contrastive_Learning/simclr1/__pycache__/utils.cpython-39.pyc and b/Contrastive_Learning/simclr1/__pycache__/utils.cpython-39.pyc differ
diff --git a/Contrastive_Learning/simclr1/io_utils.py b/Contrastive_Learning/simclr1/io_utils.py
index 594b540..4bbdacf 100644
--- a/Contrastive_Learning/simclr1/io_utils.py
+++ b/Contrastive_Learning/simclr1/io_utils.py
@@ -20,10 +20,10 @@ def parse_args(script):
     parser.add_argument('--feature_dim', default=128, type=int, help='Feature dim for latent vector')
     parser.add_argument('--temperature', default=0.5, type=float, help='Temperature used in softmax')
     parser.add_argument('--k', default=200, type=int, help='Top k most similar images used to predict the label')
-    parser.add_argument('--batch_size', default=16, type=int, help='Number of images in each mini-batch')
+    parser.add_argument('--batch_size', default=128, type=int, help='Number of images in each mini-batch')
     parser.add_argument('--epochs', default=500, type=int, help='Number of sweeps over the dataset to train')
     # parser.add_argument('--amp', action='store_true', help='引用--amp时为True，没有引用时为False')
-    parser.add_argument('--amp', default=False, type=str2bool, help='amp')
+    parser.add_argument('--amp', default=True, type=str2bool, help='amp')
     parser.add_argument('--results_path', default="results", type=str, help='results/')
     parser.add_argument('--datasets_path', default="../../../../input", type=str, help='../../input')
     parser.add_argument('--amp_level', default='O2', type=str, help='amp_level')
diff --git a/Contrastive_Learning/simclr1/main.py b/Contrastive_Learning/simclr1/main.py
index b514751..4e36833 100644
--- a/Contrastive_Learning/simclr1/main.py
+++ b/Contrastive_Learning/simclr1/main.py
@@ -29,7 +29,7 @@ import numpy as np
 from io_utils import set_seed, parse_args
 args = parse_args('main')
 
-os.environ["CUDA_VISIBLE_DEVICES"] = '7'
+os.environ["CUDA_VISIBLE_DEVICES"] = '5'
 os.environ["WANDB_API_KEY"] = 'ae69f83abb637683132c012cd248d4a14177cd36'
 os.environ['WANDB_MODE'] = args.wandb
 
